db/migrate_cards/data/metric_migration.json

```diff
@@ -0,0 +1 @@
ey}}| Add a new Source<i class=\"fa fa-caret-right\"></i>]]\r\n</div>\r\n<div class=\"right-sidebar-button button-secondary-a\">\r\n  [[/Claim?company={{_1|key}}|View all {{_1+claims+*count|core}} Claims<i class=\"fa fa-caret-right\"></i>]]\r\n</div>\r\n\r\n<div class=\"right-sidebar-button button-secondary-a\">\r\n  [[/Source?company={{_1|key}}|View all {{_1+sources+*count|core}} Sources<i class=\"fa fa-caret-right\"></i>]]\r\n</div>\r\n{{_1+values||content;structure:companypage value item}}\r\n\r\n{{#coming soon|titled;title:Coming Soon}}\r\n"},{"name":"Company+values+*type plus right+*structure","type":"Search","content":"{ \"left\":{ \"left\":{\"type\":\"metric\"}, \"right\":\"_self\" }, \"right\":{\"type\":\"year\"} }"},{"name":"import page script","type":"JavaScript","content":"wagn.slotReady(function(slot) {\r\nslot.find( \"#uncheck_all\" ).change(function(eventObject) {\r\n  if ($(this).is(\":checked\"))\r\n  {\r\n          $( \"#partial\" ).removeAttr('checked');\r\n          $( \"#exact\" ).removeAttr('checked');\r\n          slot.find(\".import_table\").find('input:checkbox').removeAttr(\"checked\");\r\n  }\r\n});\r\nslot.find( \"#partial\" ).change(function(eventObject) {\r\n  if ($(this).is(\":checked\"))\r\n  {\r\n          $( \"#uncheck_all\" ).removeAttr('checked');\r\n          slot.find(\".import_table\").find(\"tr\").each(function() {\r\n            $this = $(this);\r\n            var match = $this.find(\"td:nth-child(4)\").html();\r\n            if(match==\"partial\")\r\n                    $this.find(\"td:nth-child(1)\").find(\"input:checkbox\").prop(\"checked\",true);\r\n            \r\n          });\r\n  }\r\n});\r\nslot.find( \"#exact\" ).change(function(eventObject) {\r\n  if ($(this).is(\":checked\"))\r\n  {\r\n          $( \"#uncheck_all\" ).removeAttr('checked');\r\n          slot.find(\".import_table\").find(\"tr\").each(function() {\r\n            $this = $(this);\r\n            var match = $this.find(\"td:nth-child(4)\").html();\r\n            if(match==\"exact\")\r\n                    $this.find(\"td:nth-child(1)\").find(\"input:checkbox\").prop(\"checked\",true);\r\n            \r\n          });\r\n  }\r\n});\r\n});"},{"name":"*all+*script","type":"Pointer","content":"[[script: jquery]]\n[[script: tinymce]]\n[[script: slot]]\n[[script: card menu]]\n[[script: jquery helper]]\n[[wikirate coffee]]\n[[chosen proto script]]\n[[chosen script]]\n[[claim chosen]]\n[[claim clipboard script]]\n[[zeroclipboard]]\n[[tooltip script]]\n[[modal window script]]\n[[user profile tab script]]\n[[readmore_script]]\n[[page preview button link]]\n[[activate readmore script]]\n[[suggested source script]]\n[[empty tab content script]]\n[[source claim list script]]\n[[claim citation script]]\n[[script: filter search]]\n[[newPageJs]]\n[[newClaimJs]]\n[[homepage carousel init]]\n[[top bar shadow script]]\n[[slick js]]\n[[homepage ie8 handling script]]\n[[slick nav js]]\n[[mobile navigation js]]\n[[script: ace]]\n[[import page script]]"}]}}
```

Comment:
it is one-line format from the export now. let me try to make a pretty print on the json